[2023-01-05T10:07:01.946+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_import_data_from_apis.create_emr manual__2023-01-05T10:05:00.167541+00:00 [queued]>
[2023-01-05T10:07:01.961+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_import_data_from_apis.create_emr manual__2023-01-05T10:05:00.167541+00:00 [queued]>
[2023-01-05T10:07:01.961+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-01-05T10:07:01.961+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 3
[2023-01-05T10:07:01.961+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-01-05T10:07:01.976+0000] {taskinstance.py:1389} INFO - Executing <Task(EmrCreateJobFlowOperator): create_emr> on 2023-01-05 10:05:00.167541+00:00
[2023-01-05T10:07:01.981+0000] {standard_task_runner.py:52} INFO - Started process 157 to run task
[2023-01-05T10:07:01.985+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'atn_import_data_from_apis', 'create_emr', 'manual__2023-01-05T10:05:00.167541+00:00', '--job-id', '3702', '--raw', '--subdir', 'DAGS_FOLDER/market_data_analytics/my_dags/import_data_from_apis.py', '--cfg-path', '/tmp/tmpz2r59evl', '--error-file', '/tmp/tmpklc9knk_']
[2023-01-05T10:07:01.985+0000] {standard_task_runner.py:80} INFO - Job 3702: Subtask create_emr
[2023-01-05T10:07:02.057+0000] {task_command.py:371} INFO - Running <TaskInstance: atn_import_data_from_apis.create_emr manual__2023-01-05T10:05:00.167541+00:00 [running]> on host 370ed6868bc3
[2023-01-05T10:07:02.143+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=atn
AIRFLOW_CTX_DAG_ID=atn_import_data_from_apis
AIRFLOW_CTX_TASK_ID=create_emr
AIRFLOW_CTX_EXECUTION_DATE=2023-01-05T10:05:00.167541+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-05T10:05:00.167541+00:00
[2023-01-05T10:07:02.145+0000] {emr.py:297} INFO - Creating JobFlow using aws-conn-id: aws_credentials, emr-conn-id: aws_credentials
[2023-01-05T10:07:02.158+0000] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2023-01-05T10:07:02.166+0000] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2023-01-05T10:07:02.167+0000] {connection_wrapper.py:210} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2023-01-05T10:07:10.001+0000] {emr.py:311} INFO - JobFlow with id j-275LF5U6JZLEP created
[2023-01-05T10:07:10.063+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=atn_import_data_from_apis, task_id=create_emr, execution_date=20230105T100500, start_date=20230105T100701, end_date=20230105T100710
[2023-01-05T10:07:10.108+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-05T10:07:10.180+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
